scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+
+# Install node 20.x
+RUN apt-get update && apt-get install -y curl && \
+    curl -sL https://deb.nodesource.com/setup_20.x | bash - && \
+    apt-get install -y nodejs
+
+# Feature-parity with node.js base images.
+RUN apt-get update && apt-get install -y --no-install-recommends git ssh && \
+    npm install -g yarn
+
+# japanese font
+RUN apt install -y \
```

Hadolint

[warning] Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead

(DL3027)